server/chat-commands/core.ts

```diff
@@ -225,47 +229,42 @@ export const commands: Chat.ChatCommands = {
 
 		return target;
 	},
-	mehelp: [`/me [action] - Adds the given [action] into chat, attributed to the user.`],
+	mehelp: [
+ `/me [action] - Adds the user and attributes the optional [action] to them.`,
+ `/mee [message] - Like /me, but does not insert a space between the user and the message, and cannot start with a letter or number.`
+ ],
 
 	shrug(target) {
 		target = target ? ' ' + target + ' ' : '';
 		if (target.startsWith(' /me')) target = target.slice(1);
 		return this.checkChat(target + '¯\\_(ツ)_/¯');
 	},
-	shrughelp: ['/shrug [message] - Sends the given message, if any, appended with ¯\\_(ツ)_/¯'],
+	shrughelp: ['/shrug [message] - Sends ¯\\_(ツ)_/¯ appended to the optional message.'],
```

Comment:
This command, as well as ``/tableflip`` and ``/untableflip`` need clarifying notes regarding the command being possibly banned in some chat rooms.

It should actually be possible to figure out whether it's indeed banned or allowed in the current room, and display a different help message according to that.